docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
+
+Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes and inferred donor gender.
+
+#### File Types
+
+| Filename Pattern                | Description                                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ------------------------------- | ---------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.genotyped_gender_metrics.tgz | Generated by Sanger `compareBamGenotypes.pl` script. | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Variant Calling Supplements
```

Comment:
`Variant Calling Supplement` is a dataType under different variant call types, it contains supplementary information for variant calling, should it be under `Variant Calls` section not `Quality Control Metrics`?

Reply:
yes this makes sense! i have moved it.